main.js

```diff
@@ -6,3 +6,64 @@ JS:
 4. bonus: clicking on the translucent background closes the dialog
 5. bonus: use Class
 */
+function dialogBox($component) {
+  const questionContainer = $component.find('.question');
+  const questionContainerHtml = questionContainer.html();
+  const socialItem = $('.social-link');
+  const buttonOk = $component.find('.button--ok');
+  const buttonCancel = $component.find('.button--cancel');
+  const dialogContainer = $('.dialog');
+  let socialLink = socialItem.attr('data-url');
+
+  function init() {
+    attachDialogBox();
+    closeDialog();
+  }
+
+  init();
+
+  function attachDialogBox() {
+    socialItem.on('click', function() {
+
+      const socialText = $(this).text();
+      socialLink = $(this).attr('data-url');
+
+      questionContainer.html(questionContainerHtml.replace('{{socialNetworkName}}', socialText));
+      questionContainer.html(questionContainer.html().replace('{{socialNetworkName}}', socialText));
+      dialogContainer.removeClass('is-hidden');
+
+      buttonOk.on('click', function() {
+
```

Comment:
No `preventDefault`

Reply:
A new event is added with each click on `socialItem`. Should use `off`, or - preferably - should just declare the event outside of `socialItem.on('click')`